initial-config/static/content/wicked.css

```diff
@@ -177,6 +177,24 @@ a {
     border-color: #8a2f23
 }
 
+.btn-oauth2 {
+    color: #333;
+    background-color: #f2f2f2;
+    border-color: #ccc
+}
+
+.btn-oauth2.focus, .btn-oauth2:focus {
+    color: #333;
+    background-color: #d9d9d9;
+    border-color: #8c8c8c
+}
+
+.btn-oauth2.active, .btn-oauth2:active, .btn-oauth2:hover, .open > .dropdown-toggle.btn-oauth2 {
+    color: #333;
+    background-color: #d9d9d9;
+    border-color: #adadad
+}
+
 
```

Comment:
Ah, oh, stupid little problem here: These CSS styles would need to be merged into a user's `wicked.css` when starting either the Kickstarter or the Portal API, in an update step. Otherwise we'd use the styles with an existing static configuration, and they're not there, and things look bad. See e.g. https://github.com/Haufe-Lexware/wicked.haufe.io/blob/master/doc/development/update-globals-json.md for some info on how that needs to be done.

Reply:
Done, but please review the update step to see if it is the right thing to do